lm_eval/api/model.py

```diff
@@ -304,7 +304,9 @@ def _encode_pair(self, context, continuation):
             continuation = context[-n_spaces:] + continuation
             context = context[:-n_spaces]
 
-        if self.AUTO_MODEL_CLASS == transformers.AutoModelForCausalLM:
+        model_class = getattr(self, "AUTO_MODEL_CLASS", None)
```

Comment:
What is supposed to happen, or what is supposed to be the return value of the `_encode_pair` if `model_class` is `None`? `context_enc, continuation_enc` are not defined in this case, as far as I can see.

Reply:
You are right! For None the code fails. Fixed this)